features/docker.feature

```diff
@@ -17,7 +17,7 @@ Feature: Build docker images with pro services
 
         RUN --mount=type=secret,id=ua-attach-config \
             apt-get update \
-            && apt-get install --no-install-recommends -y ubuntu-advantage-tools ca-certificates \
+            && env DEBIAN_FRONTEND=noninteractive apt-get install -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" --no-install-recommends -y ubuntu-advantage-tools ca-certificates \
```

Comment:
this change is not needed; there may be a bug in the `| true` below though

Reply:
oops yeah I didn't even mean to push that, will fix